include/odp/api/spec/ipsec.h

```diff
@@ -1715,6 +1789,26 @@ int odp_ipsec_result(odp_ipsec_packet_result_t *result, odp_packet_t packet);
  */
 int odp_ipsec_status(odp_ipsec_status_t *status, odp_event_t event);
 
+/**
+ * IPSEC TEST API to set IPSEC SA parameter type field with new parameter value.
+ *
+ * This operation allows application to modify IPSEC SA parameters not otherwise
+ * exposed to the application. For IPSEC operations, fields like sequence number
+ * is not available to the application in the datapath. The API allows
+ * application to verify various features provided by the platform.
+ *
+ * It is the responsiblity of the application to make sure that no traffic is
+ * flowing through the SA when the API is called. Otherwise, this operation may
+ * lead to undefined behavior.
```

Comment:
How about using the following description here:

```
* IPsec test API for modifying internal state of an SA.
 *
 * This function is not meant to be used by normal applications but by special
 * test applications that test or debug the operation of the underlying ODP
 * implementation. Calling this function may degrade the performance of the
 * calling thread, other threads or the IPsec implementation in general.
 *
 * Calling this function for an SA at the same time when the SA is used for
 * processing traffic or when the SA is being modified through other parts
 * of IPsec API may result in undefined behaviour.
 *
 * SA state update through this function may not be supported by all ODP
 * implementations, ODP instances or SA instances or at every moment. This
 * function may fail for unspecified reasons even when the capability
 * call indicated support for updating a particular parameter and previous
 * similar calls succeeded.
 ```

Reply:
Ack.